nixos/modules/module-list.nix

```diff
@@ -214,6 +214,7 @@
   ./programs/partition-manager.nix
   ./programs/plotinus.nix
   ./programs/proxychains.nix
+  ./programs/qdmr.nix
```

Comment:
The commit message/title adding the NixOS module should say:

`nixos/qmdr: init`

Reply:
Can you check if the history is okay now?